### ISSUE TITLE
Accessibility: jupyter.org/community

### ISSUE THREAD

**Issue by Abolade017** [jupyterhub/outreachy#94]
### What page is this for?

https://github.com/jupyterhub/outreachy/issues/38#issuecomment-1274830800

### WAVE accessibility report

I accessed [https://jupyter.org/community](url) page. The following below is the evaluation:

![jupytercommunity](https://user-images.githubusercontent.com/70269933/195116665-2f6ec76b-895c-4c1d-a61a-02741dc46ae3.JPG)

### **Evaluation**

- **Contrast error (9) :** Very low contrast between text and background colors. Adequate contrast of text is necessary for all users, especially users with low vision. it can be fixed by increase the contrast between the foreground (text) color and the background color. Large text (larger than 18 point or 14 point bold) does not require as much contrast as smaller text.
- **Suspicious link text (1) :** Link text contains extraneous text or may not make sense out of context.Links, which are often read out of context, should clearly describe the destination or function of the link. Ambiguous text, text that does not make sense out of context, and extraneous text (such as "click here") can cause confusion and should be avoided. it can be fixed by rewording the link text so that it is more descriptive of its destination when read out of context where appropriate. Remove any extraneous text (such as "click here").
- **Redundant link (2) :** It means adjacent links go to the same `URL`. When adjacent links go to the same location (such as a linked product image and an adjacent linked product name that go to the same product page) this results in additional navigation and repetition for keyboard and screen reader users. If possible, combine the redundant links into one link and remove any redundant text or alternative text (for example, if a product image and product name are in the same link, the image can usually be given `alt=""`).
- **Alternative text (1) :** Image alternative text is present. Alternative text presents the content or function of an image to screen reader users or in other situations where images cannot be seen or are unavailable. Ensure that the alternative text conveys the content and function of the image accurately and succinctly. The alt attribute should be equivalent, accurate, and succinct.
- **Linked image with alternative (1) :** Alternative text is present for an image that is within a link. Including appropriate alternative text on an image within a link ensures that the function and purpose of the link and the content of the image is available to screen reader users or when images are unavailable. it can be fixed by ensuring that the alternative text presents the content of the image and/or the function of the link. If the full content and function of the link is presented in text within the link (an image and a text caption both within the same link, for example), then the image should generally be given empty/null alternative text (`alt=""`) to avoid redundancy.
- **Language (1) :** The language of the document or a page element is identified. Identifying the language of the page or portion of page (if different from the page itself) allows screen readers to read the content appropriately. Ensure the language is properly identified for the page (e.g., `<html lang="en">` ). If content within the page is in a language different than the page's language, identify it using a valid `lang` attribute value (e.g., `<p lang="fr">` ).
- **Heading level (8) :** this comprises of one first level heading (`<h1>` element) is present and seven heading level two (`<h2>` element) is present. Headings facilitate page navigation for users of assistive technologies. They also provide semantic and visual meaning and structure to the document. First level headings should contain the most important heading(s) on the page (generally the document title), heading(s) facilitate page navigation for users of assistive technologies. They also provide semantic and visual meaning and structure to the document. Ensure that the text in question is truly a heading and that it is structured correctly in the page outline.
- **Unordered list (5) :** An unordered (bulleted) list (`<ul>` element) is present. Ordered lists present a group of related, parallel items. Users of many assistive technologies can navigate by and within lists. it can be fixed by ensuring  that an unordered (bulleted) list is appropriate for the context. If list items are sequential or numbered, an ordered list (`<ol>`) is likely more appropriate.
- **Navigation (1) :** A `<nav>` element or navigation landmark is present. The navigation identifies a section of navigation links and can facilitate page semantics and navigation. it can be fixed by ensuring that the element defines page navigation. Multiple navigation elements on one page can be differentiated with `ARIA` labels.
- **Footer (1) :** A `<footer`> element or `contentinfo` landmark is present. Footers identify a footer for the page or a page section. It typically identifies authorship, related links, copyright date, or other footer content. Footers facilitate page semantics and navigation. it can be fixed by ensuring that the element surrounds and defines page or page section footer content.
- **ARIAL label (23) :** An `aria-label` or `aria-labelledby` attribute is present. `ARIA` labels define accessible names to be read by screen readers for interface elements. They may be used when `HTML` associations (`label`, `alternative text`, etc.) are not sufficient. it can be fixed by ensuring that the `aria-label` value or element referenced by `aria-labelledby` provides a descriptive, accurate label. When possible, use standard `HTML <label>` or other markup to make the association.

### The HTML element to be changed

```html
I will change the background-color value of 
.resource a {
  background-color: #e46e2e;
  border-radius: 2px;
  color: white;
  letter-spacing: 1px;
  padding: 13px;
  text-decoration: none;
  -webkit-transition: 0.2s;
}
```

### Your proposed HTML (or CSS), after change

```html
The background-color will be 
  .resource a {
  background-color:rgba(253, 113, 19, 0.87); 
  color:white;
 font-weight: bold;
  border-radius: 2px;
  color: white;
  letter-spacing: 1px;
  padding: 13px;
  text-decoration: none;
  -webkit-transition: 0.2s;
}
```

**Comment by minrk** [comment]
duplicate of #93
